src/model/transaction/__tests__/__snapshots__/getContentStateFragment-test.js.snap

```diff
@@ -261,7 +261,7 @@ Object {
     "data": Object {},
     "depth": 0,
     "key": "key10",
-    "nextSibling": "D",
+    "nextSibling": null,
```

Comment:
Make sure we rememeber to flag that this was a bug on the previous implementation to allow reviewers to see why this snapshot has changed